models/aliyun_ftasr_api_zh/SBI

```diff
@@ -8,11 +8,29 @@ fi
 scp=$1
 dir=$2
 oss_scp=${scp}_oss
+part_nums=10
 
 # download ossutil64 tools
 wget -O ossutil64 https://gosspublic.alicdn.com/ossutil/1.7.13/ossutil64
```

Comment:
I just realized that there might be multiple model-images running in parallel during the benchmark, downloading is not isolated, it may have side effect to other benchmarking process. Could you please:
1. use a pre-downloaded ossutil binary in model-image (as your initial submission)
2. and add a MD5 check in SBI to ensure the binary is consistent with expected MD5 signature
3. and add a comment in the code as well, telling where to find this MD5 signature(e.g. Official Aliyun webpage)